uds/uds_config_tool/FunctionCreation/ReadDataByIdentifierMethodFactory.py

```diff
@@ -9,39 +9,27 @@
 __email__ = "richard.clubb@embeduk.com"
 __status__ = "Development"
 
-
-import sys
+import logging
+from typing import Dict, List
+from xml.etree.ElementTree import Element as XMLElement
 
 from uds.uds_config_tool import DecodeFunctions
-from uds.uds_config_tool.FunctionCreation.iServiceMethodFactory import (
-    IServiceMethodFactory,
-)
+from uds.uds_config_tool.FunctionCreation.iServiceMethodFactory import \
```

Comment:
This isn't correctly formatted, please run black on the entire project

Reply:
Done.